dm_cli/utils.py

```diff
@@ -115,3 +118,36 @@ def concat_dependencies(
             raise ApplicationException(f"Conflicting dependency alias(es) in file '{filename}'. '{alias_intersect}'")
     old_dependencies.update(entity_dependencies)
     return old_dependencies
+
+
+def unpack_and_save_zipfile(override: bool, export_location: str, zip_file: ZipFile):
+    """Unpack zipfile and save it to export_location. It is assumed that zip file only contains json files and folders.
+    If override is True, any existing file/folder with the name of filename will be overwritten.
+    """
+    if ".zip" not in zip_file.filename:
+        raise ApplicationException(message="file ending .zip must be included in filename!")
+    zip_file_unpacked_path = f"{export_location}/{zip_file.filename.rstrip('.zip')}"
+
+    if not override and (Path(zip_file_unpacked_path).exists() or Path(f"{zip_file_unpacked_path}.json").exists()):
+        new_path = f"{zip_file_unpacked_path}-{str(uuid4())}"
+        os.mkdir(new_path)
+        zip_file.extractall(path=new_path)
+        # TODO fix saving of single file when override is False. If zip_file contains single file, a new folder will be created instead of a single file.
+    else:
+        zip_file.extractall(path=export_location)
+
+
+def save_as_zip_file(override: bool, export_location: str, filename: str, data: str):
+    """Save binary data into a zip file on the local disk.
+    If override is True, any existing zip file with the name of filename will be overwritten.
+    """
+    if ".zip" not in filename:
```

Comment:
```suggestion
		# Should probably check that it ends with the .zip extension, not that it contains it?
    if not filename.endswith(".zip"):
    # alternatively, use pathlib:
    # from pathlib import Path
    if not Path(filename).suffix == ".zip":
    
```
Here, and everywhere else if agreed upon

Reply:
agree, endswith() would be nice here